packages/sit-onyx/src/components/OnyxTooltip/types.ts

```diff
@@ -28,6 +28,10 @@ export type OnyxTooltipProps = DensityProp & {
    * How to position the tooltip relative to the parent element.
    */
   position?: TooltipPosition;
+  /**
+   * How to float the tooltip relative to the parent element.
+   */
+  float?: TooltipFloat;
```

Comment:
```suggestion
  alignment?: TooltipFloat;
```

I think alignment is more suitable :)